src/dataIntegration/transcriptomics/FASTCORE/fastcore.m

```diff
@@ -45,7 +45,7 @@
     model_orig = model;
 
     %Find irreversible reactions
-    irrevRxns = find(model.rev>=0);
+    irrevRxns = find(model.rev==0);
```

Comment:
May you also keep the header of the function that you already formatted? Thanks 😄